crates/rattler_lock/src/parse/models/v6/conda_package_data.rs

```diff
@@ -157,6 +157,7 @@ impl<'a> TryFrom<CondaPackageDataModel<'a>> for CondaPackageData {
             build_number,
             constrains: value.constrains.into_owned(),
             depends: value.depends.into_owned(),
+            optional_depends: std::collections::BTreeMap::new(),
```

Comment:
I think you should also add this to the serialization logic so that we also have optional dependencies in the lock-files!